library/ajax/graphs.php

```diff
@@ -267,107 +222,54 @@ function graphsGetRanges($name) {
      // no conversion, so use raw value
      $y=$row["$name"];
     }
-    $data[] = new scatter_value($x, $y);
+    $data[$x][$name] = $y;
   }
 }
-$s->set_values( $data );
 if ($isBP) {
   //set up the other blood pressure line
-  $data = array();
   while ($row = sqlFetchArray($values_alt)) {
     if ($row["$name_alt"]) {
-      $x=$row['unix_date'];
-    if ($multiplier) {
-      // apply unit conversion multiplier
-      $y=$row["$name_alt"]*$multiplier;
-    }
-    else if ($isConvertFtoC ) {
-      // apply temp F to C conversion
-      $y=convertFtoC($row["$name_alt"]);
-    }
-    else {
-     // no conversion, so use raw value
-     $y=$row["$name_alt"];
-    }
-      $data[] = new scatter_value($x, $y);
+      $x=$row['date'];
+      if ($multiplier) {
+        // apply unit conversion multiplier
+        $y=$row["$name_alt"]*$multiplier;
+      }
+      else if ($isConvertFtoC ) {
+        // apply temp F to C conversion
+        $y=convertFtoC($row["$name_alt"]);
+      }
+      else {
+       // no conversion, so use raw value
+       $y=$row["$name_alt"];
+      }
```

Comment:
this was fixing the spacing. note original had no indent after if ($row["$name_alt"]) {